Mu2eG4/src/ConstructMaterials.cc

```diff
@@ -259,8 +259,8 @@ namespace mu2e {
     mat = uniqueMaterialOrThrow( "IPAPolyethylene");
     {
       G4Material* IPAPolyethylene = new G4Material( mat.name, 0.96*CLHEP::g/CLHEP::cm3, 2);
-      IPAPolyethylene->AddMaterial( findMaterialOrThrow("G4_H"), 0.14);
-      IPAPolyethylene->AddMaterial( findMaterialOrThrow("G4_C"), 0.86);
+      IPAPolyethylene->AddMaterial( findMaterialOrThrow("G4_H"), 0.12);
+      IPAPolyethylene->AddMaterial( findMaterialOrThrow("G4_C"), 0.88);
```

Comment:
Polyethylene has 2:1 hydrogen:carbon molar ratios.  Then the old hydrogen mass fraction 0.14 is correct and the new one is wrong.   What am I missing?

Reply:
According to Hank Glass the poly used for the IPA has ~2% carbon doping to make it conductive.  Note that the previous density was wrong.

Reply:
OK, this answers my question.